apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -34,7 +45,50 @@ import { ApiResponse } from "@calcom/platform-types";
 export class BookingsController {
   private readonly logger = new Logger("ee bookings controller");
 
-  constructor(private readonly oAuthFlowService: OAuthFlowService) {}
+  constructor(
+    private readonly oAuthFlowService: OAuthFlowService,
+    private readonly prismaReadService: PrismaReadService
+  ) {}
+
+  // note(Rajiv): currently this endpoint is atoms only
+  @Get("/")
+  @UseGuards(AccessTokenGuard)
+  async getBookings(
+    @GetUser() user: User,
+    @Query() queryParams: GetBookingsInput
+  ): Promise<ApiResponse<unknown>> {
+    const { filters, cursor, limit } = queryParams;
+    const bookings = await getAllUserBookings({
+      bookingListingByStatus: filters.status,
+      skip: cursor ?? 0,
+      take: limit ?? 10,
+      filters: filters,
```

Comment:
FYI: if key and variable are the same "filters: filters" you can just write:

...
take,
filters,
...